Package.swift

```diff
@@ -3,21 +3,23 @@
 import PackageDescription
 import Foundation
 
-var swiftSyntaxUnsafeSwiftFlags: [String] = []
-
 /// If we are in a controlled CI environment, we can use internal compiler flags
 /// to speed up the build or improve it.
-if ProcessInfo.processInfo.environment["SWIFT_BUILD_SCRIPT_ENVIRONMENT"] != nil {
+let swiftSyntaxSwiftSettings:[SwiftSetting] 
+if case _? = ProcessInfo.processInfo.environment["SWIFT_BUILD_SCRIPT_ENVIRONMENT"] {
```

Comment:
Is there a reason why you don’t want to use `!= nil`? IMO it’s easier to read than the pattern matching syntax. If there is a reason I would be curious to know it.
```suggestion
if ProcessInfo.processInfo.environment["SWIFT_BUILD_SCRIPT_ENVIRONMENT"] != nil {
```

Reply:
it’s clearer about the intent, since it doesn’t call into the `Equatable` implementation of `Optional<T>`. in this case, we are unwrapping the optional, but don’t care about the result

Reply:
Hmm, I see where you’re coming from but still think that the benefits of `!= nil` (I can visually parse it in <1s whereas my brain takes a couple of seconds to understand what `case _? =` is doing) outweigh the benefits not calling into the `Equatable` implementation of `Optional` (which are very, very low; if that is broken I assume almost nothing will work in Swift). 
`!=` also matches how we check for optionality of `Optional` in the rest of the project. Could you change it?

Reply:
done